android/java/res/layout/wallet_coin_list_item.xml

```diff
@@ -69,6 +69,14 @@
             android:textColor="@color/wallet_secondary_text_color"
             android:layout_gravity="end"/>
 
+        <CheckBox
+            android:id="@+id/assetCheck"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            android:layout_alignParentRight="true"
```

Comment:
layout_alignParentRight is the layout params for relative layouts. i think we should use gravity instead.

Reply:
we don't need it at all actually